src/appPackageManifest.test.ts

```diff
@@ -277,5 +285,28 @@ it("doesn't include tile data if app type is not APP", () => {
     appType: AppType.CLOCKFACE,
   } as ClockProjectConfiguration;
 
-  return expectValidPackageManifest({ projectConfig }).toMatchSnapshot();
+  return expectValidPackageManifest({
+    projectConfig,
+    nativeApp: true,
+  }).toMatchSnapshot();
+});
+
+it('it includes tiles just for native apps', () => {
```

Comment:
super nit: no need to include `it` in the name of the test too

Reply:
Made the change. Nice catch!